<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tracer_box</class>
 <widget class="QDialog" name="tracer_box">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="terminal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>278</y>
     <width>751</width>
     <height>186</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="termianl_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>318</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Terminal de entrada e saída de dados</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>741</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Tipo de dispositivo</string>
      </property>
      <widget class="QRadioButton" name="pinos_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>2 terminais</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="pinos_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>3 terminais</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>0</y>
         <width>421</width>
         <height>119</height>
        </rect>
       </property>
       <property name="title">
        <string>Tensões de base (V)</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>36</y>
          <width>406</width>
          <height>56</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="vbmin_label">
           <property name="text">
            <string>Vb Mín.</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="vbmax_label">
           <property name="text">
            <string>Vb Máx.</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="passo_label">
           <property name="text">
            <string>Passo de Vb entre curvas</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QDoubleSpinBox" name="vbmin">
           <property name="minimum">
            <double>-10.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="vbmax">
           <property name="minimum">
            <double>-10.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleSpinBox" name="vbstep"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>152</y>
     <width>741</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="caminho_label">
    <property name="text">
     <string>Caminho de saída</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="caminho"/>
     </item>
     <item>
      <widget class="QPushButton" name="but_buscar">
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="but_sobre">
      <property name="text">
       <string>Sobre</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="but_ajuda">
      <property name="text">
       <string>Ajuda</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>470</y>
     <width>246</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="but_iniciar">
      <property name="text">
       <string>Iniciar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="but_graficos">
      <property name="text">
       <string>Exibir gráficos</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
